spec/lex.dd

```diff
@@ -397,8 +397,9 @@ r"ab\n" // string is 4 characters,
 ---------------
 `hello`
 `c:\root\foo.exe`
-`ab\n`  // string is 4 characters,
-        // 'a', 'b', '\', 'n'
+`The "lazy" dog`
+`a"b\n`  // string is 5 characters,
+        // 'a', '"', 'b', '\', 'n'
 ---------------
```

Comment:
Then it would be incorrect!

```d
import std.stdio;

void main()
{
  writeln(`ab\n`);
  writeln(`ab\\n`);
}
```

```bash
giles@tikka:~$ ./test
ab\n
ab\\n
```